app/lib/feed_manager.rb

```diff
@@ -301,6 +301,35 @@ def populate_home(account)
     end
   end
 
+  def populate_list(list)
```

Comment:
I dont think you intended to commit this?

Reply:
oops